finality-aleph/src/nodes.rs

```diff
@@ -149,12 +147,18 @@ where
         genesis_header,
     );
     let finalizer = AlephFinalizer::new(client.clone(), metrics.clone());
-    let (sync_service, justifications_for_sync, _) = match SyncService::new(
+
+    let (sync_service, justifications_for_sync, _): (
+        SyncService<Block, _, _, _, _, _, _, Box<dyn ImportQueueService<Block>>>,
```

Comment:
Why is the second template arg necessary?

If it's because the inner `dyn` depends on `Block` then this is another argument for a wrapper at least, as mentioned in the previous comment.

Reply:
turns out both were not necessary...
fixed